lib/techs/tmpl-spec.js

```diff
@@ -98,6 +96,7 @@ module.exports = require('enb/lib/build-flow').create()
                                     var langTarget = target.replace('.js', '.' + lang + '.js');
                                     return {
                                         name: lang,
+                                        isAsync: engine.options.isAsync,
```

Comment:
`engine.options` contains options for `tech`, it's better to use `engine.async` instead of `engine.options.isAsync`